src/h5web/vis-packs/core/complex/MappedComplexVis.tsx

```diff
@@ -83,11 +87,19 @@ function MappedPhaseMapVis(props: Props) {
 
   return (
     <HeatmapVis
-      dataArray={phaseArray}
-      domain={safePhaseDomain}
-      alphaArray={amplitudeArray}
-      alphaDomain={amplitudeDomain}
-      title={title}
+      dataArray={dataArray}
+      domain={safeDomain}
+      alphaArray={
+        visType === ComplexVisType.PhaseAndAmplitude
+          ? amplitudeArray
+          : undefined
+      }
+      alphaDomain={
+        visType === ComplexVisType.PhaseAndAmplitude
+          ? amplitudeDomain
+          : undefined
+      }
+      title={`${title || ''} (${visType.toLowerCase()})`}
```

Comment:
Seems to me like this logic should be a bit more complex: if no `title` is present, there's a leading space and the vis type starts with a lower case.